x-pack/plugins/cases/public/components/create/connector.test.tsx

```diff
@@ -31,6 +31,16 @@ jest.mock('../connectors/resilient/use_get_severity');
 jest.mock('../connectors/servicenow/use_get_choices');
 jest.mock('../../containers/configure/use_configure');
 
+const mockUseCasesContext = jest.fn().mockReturnValue({
+  permissions: {
+    connectors: true,
+  },
+});
+
+jest.mock('../cases_context/use_cases_context', () => ({
+  useCasesContext: () => mockUseCasesContext(),
+}));
```

Comment:
I think you can use the `appMockRender` to define the permission without mocking the `CasesContext`.